README.md

```diff
@@ -348,6 +348,19 @@ You can use the  `in`  function to determine whether a property's value is one o
 // Matches all notes where the text is one of the elements in the specified list.
 let predicate = (\Note.text).in("a", "b", "c", "d")
 ```
+or pass in an Array
+
+```swift
+// Matches all notes where the text is one of the elements in the specified list.
+let predicate = (\Note.text).in(["a", "b", "c", "d"])
+```
+
+or pass in a Set 
+
+```swift
+// Matches all notes where the text is one of the elements in the specified list.
+let predicate = (\Note.text).in(Set(["a", "b", "c", "d"]))
+```
```

Comment:
Small tweak to your changes. Let's use the following for the `in` section of the documentation.

###### in

You can use the  `in`  function to determine whether a property's value is one of the values in a comma-separated list, an array, or a set.


```swift
// Matches all notes where the text is one of the elements in the specified comma-separated list.
 let predicate = (\Note.text).in("a", "b", "c", "d")

// Matches all notes where the text is one of the elements in the specified array.
let predicate = (\Note.text).in(["a", "b", "c", "d"])

// Matches all notes where the text is one of the elements in the specified set.
let predicate = (\Note.text).in(Set(["a", "b", "c", "d"]))
```

When the property is of type `String`, `in` accepts a second parameter that determines how the string should be compared to the elements in the list.

```swift
// Case-insensitive comparison.
let predicate = (\Note.text).in(["a", "b", "c", "d"], .caseInsensitive)
```